package.json

```diff
@@ -60,9 +60,6 @@
     "sinon-chai": "^2.8.0",
     "wcag": "^0.2.2"
   },
-  "optionalDependencies": {
-    "player": "^0.6.1"
```

Comment:
This'll break some other code, but I'll push up a PR to take the player out :sob: